proof-systems-shim/src/lib.rs

```diff
@@ -0,0 +1,18 @@
+// Copyright 2020 ChainSafe Systems
+// SPDX-License-Identifier: Apache-2.0
+
+//! This is a shim crate that directly reexports crates under proof-systems.
+//! The idea is to manage those crates altogether,
+//! to make it easier to coordiate rev/version updates.
+//!
+//! Other crates in this repo should always depend on this shim crate instead of directly
+//! depending on crates from proof systems
+//!
+//! ```
+//! use proof_systems::*;
+//! use mina_hasher::Hashable;
+//! ```
+//!
+
+pub use commitment_dlog;
+pub use mina_hasher;
```

Comment:
This is a good idea, It might add another dep if people want to use our crates standalone but I think it is worth it. Also not sure if shim is the right name since it isn't modifying the API at all. I don't have any better ideas though

Reply:
Regarding naming, shim is only part of the directory name but the crate is named as `proof-systems`